session/session.go

```diff
@@ -1226,6 +1227,12 @@ func CreateSession(store kv.Storage) (Session, error) {
 	}
 	privilege.BindPrivilegeManager(s, pm)
 
+	bm := &infobind.BindManager{
+		Handle:        do.BindHandle(),
+		SessionHandle: infobind.NewHandle(),
```

Comment:
The new handle will create a map of capacity 1000, I think it is not a good choice for a session handle.

Reply:
update capacity to 10?

Reply:
Yes, I think it is fine. And what about creating the map in a lazy way, for example, do not create the map when we create the handle? Since most sessions won't create the bindings.

Reply:
I update the capacity to 0, but we need a empty map. The next pull request will add bind check when a query request in, so we need a map to avoid add write lock.